src/GenericAccessRequestForm/Constants/FormSpecificConstants.ts

```diff
@@ -0,0 +1,50 @@
+const StudyRegistrationAccessRequest = {
+  name: 'StudyRegistration',
+  title: 'Study Registration Access Request',
+  description:
+    'Please fill out this form to request and be approved for access to register your study with the HEAL Platform.',
+  resultTitle: 'Your access request has been submitted!',
+  resultSubtitle:
+    'Thank you for your submission. Requests take up to 1 business day to complete. You will be notified of the status.',
+  showStudyName: true,
+  showGrantNumber: false,
+  showDisclaimer: true,
+  subjectLine: 'Study registration access request for',
+};
+const DataDictionarySubmissionRequest = {
+  name: 'DataDictionarySubmissionRequest',
+  title: 'Data Dictionary Submission Request',
+  description:
+    'Please fill out this form to request access to submit data dictionaries to your study on the HEAL Platform.',
+  resultTitle: 'Your access request has been submitted!',
+  resultSubtitle:
+    'Thank you for your submission. Requests take up to 1 business day to complete. You will be notified of the status.',
+  showStudyName: true,
+  showGrantNumber: false,
+  showDisclaimer: true,
+  subjectLine: 'Data dictionary submission access request for',
+};
+const WorkSpaceRegister = {
+  name: 'WorkSpaceRegister',
```

Comment:
nitpicking: they should really be `WorkspaceAccessRequest` (this line and the line above it)